src/main/kotlin/com/temetra/vroomapi/RouteController.kt

```diff
@@ -129,6 +130,12 @@ class RouteController : ErrorController {
             progArgs.add("-g")
         }
 
+        if(cluster == "EU"){
```

Comment:
Isn't it better to have a switch (in Kotlin is called when) expression, with each exception and a default value for everything else? see the example on their documentation https://kotlinlang.org/docs/control-flow.html#when-expression